### PR TITLE
Vendor in latest containers image and storage and drop testing on golang 1.8.

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@pixdrift @nalind @TomSweeneyRedHat @umohnani8 @giuseppe PTAL

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@ language: go
 dist: trusty
 sudo: required
 go:
-    - 1.8
```
Comment (TomSweeneyRedHat) [inline .travis.yml]: should we add 1.11 yet?

File: vendor.conf [hunk 2]
```diff
@@ -4,8 +4,8 @@ github.com/BurntSushi/toml master
 github.com/containerd/continuity master
 github.com/containernetworking/cni v0.6.0
 github.com/seccomp/containers-golang master
-github.com/containers/image master
-github.com/containers/storage afdedba2d2ad573350aee35033d4e0c58fdbd57b
+github.com/containers/image 134f99bed228d6297dc01d152804f6f09f185418
+github.com/containers/storage 17c7d1fee5603ccf6dd97edc14162fc1510e7e23
```
Comment (TomSweeneyRedHat) [inline vendor.conf]: Education time for me.  Why do you want to go to a version rather than master for either or both of these?

**Comment by TomSweeneyRedHat** [comment]
nit of a nit on the title.  "lates" to "latest" if you get a chance, if not, no worries.

**Comment by rhatdan** [comment]
We might need to sick @baude on Buildah to bring our test times down as well.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**